app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
```

Comment:
You don't really need this. Take a look at what we are doing in Portus: https://github.com/SUSE/Portus/blob/master/app/assets/javascripts/application.js